org.eclipse.jdt.core/.settings/.api_filters

```diff
@@ -1,27 +1,26 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <component id="org.eclipse.jdt.core" version="2">
-    <resource path="META-INF/MANIFEST.MF">
-        <filter id="924844039">
+    <resource path="META-INF/MANIFEST.MF" type="org.eclipse.jdt.core.dom.EnhancedForWithRecordPattern">
+        <filter id="305324134">
             <message_arguments>
-                <message_argument value="3.38.0"/>
-                <message_argument value="3.38.0"/>
+                <message_argument value="org.eclipse.jdt.core.dom.EnhancedForWithRecordPattern"/>
+                <message_argument value="org.eclipse.jdt.core_3.38.0"/>
             </message_arguments>
         </filter>
     </resource>
-    <resource path="META-INF/MANIFEST.MF" type="org.eclipse.jdt.core.dom.EnhancedForWithRecordPattern">
-        <filter id="305324134">
+    <resource path="META-INF/MANIFEST.MF" type="org.eclipse.jdt.core.dom.UnnamedClass">
+        <filter comment="@noreference was missed out on the preview feature" id="305324134">
             <message_arguments>
-                <message_argument value="org.eclipse.jdt.core.dom.EnhancedForWithRecordPattern"/>
+                <message_argument value="org.eclipse.jdt.core.dom.UnnamedClass"/>
                 <message_argument value="org.eclipse.jdt.core_3.38.0"/>
             </message_arguments>
         </filter>
     </resource>
-    <resource path="dom/org/eclipse/jdt/core/dom/AST.java" type="org.eclipse.jdt.core.dom.AST">
-        <filter id="388194388">
+    <resource path="dom/org/eclipse/jdt/core/dom/ASTMatcher.java" type="org.eclipse.jdt.core.dom.ASTMatcher">
+        <filter comment="Preview Feature - Class name changed - should not have been an API in the first place " id="338792546">
             <message_arguments>
-                <message_argument value="org.eclipse.jdt.core.dom.AST"/>
```

Comment:
I see API error in IDE: `The constant value 21 of the field org.eclipse.jdt.core.dom.AST.JLS_Latest has been changed`.